README.md

```diff
@@ -18,57 +20,42 @@ To hash a list of bytes, invoke the [`convert`][convert] method on the
 objects.
 
 ```dart
-import 'package:crypto/crypto.dart';
 import 'dart:convert'; // for the UTF8.encode method
+import 'package:crypto/crypto.dart';
```

Comment:
This should be separated from the `dart:` import with a newline. See [the style guide](https://www.dartlang.org/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports).

Also below.

Reply:
ack